notebooks/wikiqa/arci.ipynb

```diff
@@ -734,7 +734,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.3"
+   "version": "3.6.5"
```

Comment:
do not commit useless changes.

Reply:
I will recover this file to the previous version.

Reply:
All the notebooks would be modified as we will tune the parameters~